nemo_skills/pipeline/utils/cluster.py

```diff
@@ -106,6 +106,48 @@ def get_timeout_str(cluster_config, partition, with_save_delay: bool = True) ->
     return timeout_str
 
 
+def parse_extra_metrics_arguments(extra_metrics_arguments: str | dict | None) -> dict | None:
```

Comment:
let's maybe have a shared function that will be used both here and for sbatch kwargs (and any other future use-cases where we need to take a dict as an argument)? In fact there is already another one with `extra_judge_pipeline_args` in eval.py which we can use this function for